sdk/communication/tests.yml

```diff
@@ -1,5 +1,17 @@
 trigger: none
 
+parameters:
+  - name: TestPackagesEnabled # by overriding this parameter, developers can control what tests to run
+    displayName: Test Packges Enabled
```

Comment:
I actually ended up changing this to "Tests Enabled" as the friendly display name

Reply:
will change it to keep them consistent